pages/table/compact-mode.page.tsx

```diff
@@ -0,0 +1,310 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This is a great page for demos, but it's not currently used for any integration tests. Do we really need to maintain this big page? Maybe we can just add the content density preferences to our existing table pages?

Reply:
I added this preference to the [simple page](https://github.com/cloudscape-design/components/pull/748/files#diff-8917415c3038e39f2943e6721572c693f1e3183331ebe4637ee8d033b7bed8e6R35).

I can still see some value on having screenshot tests on this page (all tables are "compact" by default) and I can add a couple of integration tests for checking if toggling the content density mode does change the elements.